newsfragments/3408.deprecation.rst

```diff
@@ -0,0 +1,4 @@
+Deprecated ``web3.utils.abi`` utils and types.
```

Comment:
This newsfragment pattern doesn't display as I _think_ was intended, based on the format that's here. We should either revise it or accept that it will be one sentence followed by a text block or one sentence after another.